src/clp_ffi_py/Py_utils.cpp

```diff
@@ -2,24 +2,29 @@
 
 #include "Py_utils.hpp"
 
+#include <string>
 #include <string_view>
 
+#include <clp/ir/types.hpp>
+
 #include <clp_ffi_py/PyObjectCast.hpp>
 #include <clp_ffi_py/PyObjectUtils.hpp>
+#include <clp_ffi_py/utils.hpp>
 
 namespace clp_ffi_py {
 namespace {
-constexpr char const* const cPyFuncNameGetFormattedTimestamp{"get_formatted_timestamp"};
-PyObjectStaticPtr<PyObject> Py_func_get_formatted_timestamp{nullptr};
-
-constexpr char const* const cPyFuncNameGetTimezoneFromTimezoneId{"get_timezone_from_timezone_id"};
-PyObjectStaticPtr<PyObject> Py_func_get_timezone_from_timezone_id{nullptr};
-
+constexpr std::string_view cPyFuncNameGetFormattedTimestamp{"get_formatted_timestamp"};
+constexpr std::string_view cPyFuncNameGetTimezoneFromTimezoneId{"get_timezone_from_timezone_id"};
 constexpr std::string_view cPyFuncNameSerializeDictToMsgpack{"serialize_dict_to_msgpack"};
-PyObjectStaticPtr<PyObject> Py_func_serialize_dict_to_msgpack{nullptr};
-
 constexpr std::string_view cPyFuncNameParseJsonStr{"parse_json_str"};
-PyObjectStaticPtr<PyObject> Py_func_parse_json_str{nullptr};
+
+// NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
+PyObjectStaticPtr<PyObject> py_func_get_formatted_timestamp{nullptr};
```

Comment:
I seem to get an error about no header providing PyObject.

Reply:
which Python version were you using? `PyObject` is defined in `object.h` and it's already included in our `Python.hpp`

Reply:
yeah, I was doing linting in Python3.10